src/sentry/static/sentry/app/views/alerts/rules/details/timeline.tsx

```diff
@@ -247,74 +214,67 @@ type Props = {
 
 class Timeline extends React.Component<Props> {
   renderEmptyMessage = () => {
-    return (
-      <EmptyStateWarning small withIcon={false}>
-        {t('No alerts have been triggered yet')}
-      </EmptyStateWarning>
-    );
+    return <p>{t('No alerts triggered during this time')}</p>;
```

Comment:
With this it looks like:
![Screen Shot 2021-03-12 at 2 31 27 PM](https://user-images.githubusercontent.com/15015880/111005425-a2a78d00-833f-11eb-8475-b4197872f999.png)

I think it should be more like the other empty messages:
![Screen Shot 2021-03-12 at 2 32 04 PM](https://user-images.githubusercontent.com/15015880/111005467-b94de400-833f-11eb-89eb-831aef10ee67.png)

Reply:
I can create a small suggestion pr for this. See: https://github.com/getsentry/sentry/pull/24433